src/libraries/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs

```diff
@@ -732,14 +751,39 @@ public object GetContextInfo()
                 } while (context.IsCumulative);
             }
 
-            if (!totalOffset.IsEmpty)
+            if (!totalOffset.IsEmpty())
             {
                 // We need now to reset the total transform in the region so when calling Region.GetHRgn(Graphics)
                 // the HRegion is properly offset by GDI+ based on the total offset of the graphics object.
-                cumulClip.Translate(-totalOffset.X, -totalOffset.Y);
+
+                // If we don't have a cumulative clip, we're infinite, and translation on infinite regions is a no-op.
+                cumulativeClip?.Translate(-totalOffset.X, -totalOffset.Y);
             }
+        }
+
+        /// <summary>
+        ///  Gets the cumulative offset.
+        /// </summary>
+        /// <param name="offset">The cumulative offset.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void GetContextInfo(out PointF offset)
+        {
+            GetContextInfo(out Matrix3x2 cumulativeTransform, calculateClip: false, out _);
+            Vector2 translation = cumulativeTransform.Translation;
+            offset = new PointF(translation.X, translation.Y);
+        }
 
-            return new object[] { cumulClip, cumulTransform };
+        /// <summary>
+        ///  Gets the cumulative offset and clip region.
+        /// </summary>
+        /// <param name="offset">The cumulative offset.</param>
+        /// <param name="clip">The cumulative clip region or null if the clip region is infinite.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
These should be annotated as `[SupportedOS("Windows")]` right?

Reply:
Ah, found it. `[SupportedOSPlatform("windows")]`.  Updating.